packages/appstore/src/components/main-title-bar/__tests__/index.spec.tsx

```diff
@@ -4,9 +4,7 @@ import { StoreProvider, mockStore } from '@deriv/stores';
 import MainTitleBar from '..';
 
 describe('MainTitleBar', () => {
-    const render_container = () => {
-        const mock = mockStore({});
-
+    const render_container = (mock: any) => {
```

Comment:
```suggestion
    const render_container = (mock: any) => {
```

Can we pass other type instead of any?

Reply:
@niloofar-deriv good catch. I almost forgot about it. added the proper type to it :+1: